src/crypto/PersistentStorageOperationalKeystore.h

```diff
@@ -64,16 +73,7 @@ class PersistentStorageOperationalKeystore : public Crypto::OperationalKeystore
         return CHIP_NO_ERROR;
     }
 
-    /**
-     * @brief Finalize the keystore, so that subsequent operations fail
-     */
-    void Finish()
-    {
-        VerifyOrReturn(mStorage != nullptr);
-
-        ResetPendingKey();
-        mStorage = nullptr;
-    }
+    void Finish();
```

Comment:
The keystore refactoring should probably also be a separate PR (not least because a different set of reviewers from the OpenSSL bits might be needed here).